Resources/Audio/Voice/Human/attributions.yml

```diff
@@ -0,0 +1,4 @@
+- files: ["female_sigh.ogg", "male_sigh.ogg"]
+  license: "CC-BY-4.0"
```

Comment:
Link says:

> Any files located in the Paradise/goon, Paradise/icons/goonstation, or Paradise/sound/goonstation directories, or any subdirectories of mentioned directories are licensed under the Creative Commons 3.0 BY-NC-SA license (https://creativecommons.org/licenses/by-nc-sa/3.0)

> All other assets including icons and sound files are licensed under the Creative Commons 3.0 BY-SA license (https://creativecommons.org/licenses/by-sa/3.0/), unless otherwise indicated.

So wouldn't be 4.0